.travis.yml

```
@@ -44,6 +46,10 @@ script:
  # Remove to enable build of sdist.
  - rm -f cabal.project
  - (cd tests; cabal v2-configure; cabal v2-test)
+ # Build the benchmarks and run a single iteration.
+ - if [[ "$GHCVER" > 7.9 ]]; then
```

Maybe I should make it clearer that this is a lexicographic comparison, intended to filter out all GHC version < 8.

```suggestion
 - if [[ "$GHCVER" > 7z ]]; then
```

---

I suggest we still a trick from `haskell-ci`:
```yaml
  - "HCNUMVER=$(${HC} --numeric-version|perl -ne '/^(\\d+)\\.(\\d+)\\.(\\d+)(\\.(\\d+))?$/; print(10000 * $1 + 100 * $2 + ($3 == 0 ? $5 != 1 : $3))')"
  - echo $HCNUMVER
  - if [ $HCNUMVER -ge 80000 ] ; then ...
```

---

Done! This made it simpler to run the benchmarks with GHC 7.10 too. ;)